src/components/AboutModal/AboutModal.js

```diff
@@ -51,8 +51,8 @@ AboutModal.propTypes = {
   show: PropTypes.bool.isRequired,
   /** Function to call when modal is closed */
   onHide: PropTypes.func.isRequired,
-  /** Text to show for the product title */
-  productTitle: PropTypes.string,
+  /** Text or Element to show for the product title */
+  productTitle: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
I think it can be as simple as:
```jsx
productTitle: PropTypes.node,
```

Since `node` can be a `string`.

Reply:
Thanks. Updated.